src/redis_index.c

```diff
@@ -462,6 +462,20 @@ int Redis_DeleteKey(RedisModuleCtx *ctx, RedisModuleString *s) {
   return res;
 }
 
+int Redis_DeleteKeyC(RedisModuleCtx *ctx, char *cstr, int replicate) {
+  RedisModuleCallReply *rep;
+  if (!replicate) {
+    rep = RedisModule_Call(ctx, "DEL", "c", cstr);
+  } else {
+    // for CRDT we need to send command to all replicas. 
+    rep = RedisModule_Call(ctx, "DEL", "c!", cstr);
```

Comment:
We need to verify if the ctx used here is the client ctx so it will automatically be wrapt with multi exec.

Reply:
The `ctx` is the one we get with `FT.DROPINDEX`. Am I missing something?

Reply:
If its the same ctx we get with the command then we are good.